javacutil/src/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -98,12 +99,15 @@ public static boolean areSame(
             return false;
         }
 
-        Map<? extends ExecutableElement, ? extends AnnotationValue> elval1 =
-                getElementValuesWithDefaults(a1);
-        Map<? extends ExecutableElement, ? extends AnnotationValue> elval2 =
-                getElementValuesWithDefaults(a2);
+        // This commented implementation is less efficient.  It is also wrong:  it requires a
+        // particular order for fields, and it distinguishes the long constants "33" and "33L".
```

Comment:
It's good to have these comments here for the review. But for the final PR they should be removed.

Reply:
I want to retain them.  They give insight into what was wrong with the old implementation and why we went to the trouble to change it.  They caution against someone reinstating the old implementation.